### ISSUE TITLE
After the 0.3.3 release, lean `find` results no longer include virtual properties

### ISSUE THREAD

**Issue by Vayvala** [vkarpov15/mongoose-lean-virtuals#13]
Virtual Property:
```
schema.virtual('startDateF').get(function() {
  return moment(this.startDate).format(DATE_FORMAT);
});
```
Execution:
```Subscription.find(query).lean({virtuals: true}).exec();```

Output does not include a `startDateF` *virtual* property

Note: it works in 0.3.2

**Comment by kikill95** [comment]
@vkarpov15 I think I know where the issue is: https://github.com/vkarpov15/mongoose-lean-virtuals/commit/ae35cbcc99423380b1f4cbd435fbd6f4c0de1d3b#diff-168726dbe96b3ce427e7fedce31bb0bcR39

that `return` allow make only 1 iteration

**Comment by artemkobets** [comment]
Created a pull request for this issue #14

**Comment by vkarpov15** [comment]
Fixed in 0.3.4 with #14: http://plugins.mongoosejs.io/plugins/lean-virtuals#0-3-4-2018-11-13

**Comment by vkarpov15** [comment]
Thanks @artemkobets
